common/minimodel/__init__.py

```diff
@@ -389,9 +390,9 @@ class TooType(IntEnum):
 
 
 class SetupTimeType(Enum):
-    FULL = auto()
-    REACQ = auto()
-    NONE = auto()
+    FULL = 'FULL'
+    REACQ = 'REACQ'
+    NONE = 'NONE'
```

Comment:
I don't think it is ever needed to have an `Enum` with its names equal to its entries, as mentioned above.